.github/workflows/deploy-prd-ecs.yml

```diff
@@ -58,9 +45,8 @@ jobs:
         run: |
           cd infrastructure/ecs-main
           npm ci
-          sls deploy --stage prd
```

Comment:
I would wait to change the `prd` deploy actions until we've deployed the code change and verified that behavior is as expected in production